pom.xml

```diff
@@ -36,10 +36,12 @@
 
   <properties>
     <spark32.version>3.2.2</spark32.version>
+    <spark32.scala>2.12.15</spark32.scala>
```

Comment:
maybe only using `scala.version` is enough, and change the version for each spark version in the profiles.

Reply:
spark master already bumps to 2.12.17(https://github.com/apache/spark/pull/37892),

Reply:
Spark 3.3 still uses 2.12.15, and also can set the different version in the each spark version profile

Reply:
I add a property <spark33.scala>